packages/ts-migrate-plugins/tests/src/explicit-any.test.ts

```diff
@@ -1,26 +1,26 @@
 import ts from 'typescript';
-import { mockPluginParams, mockDiagnostic } from '../test-utils';
+import { mockPluginParams, mockDiagnostic, realPluginParams } from '../test-utils';
 import explicitAnyPlugin from '../../src/plugins/explicit-any';
 
 describe('explicit-any plugin', () => {
   it('adds explicit any', async () => {
-    const text = `import specifier1 from '../source';
+    const text = `let somePromise: any;
 somePromise.then(res1 => res1.default || res1);
 somePromise.then((res2) => res2.default || res2);
+let someArray: any;
 someArray.forEach(({ arg1, arg2 }) => {});
-const { dest1, dest2 } = obj;
 function fn1(p1, p2) {}
 const fn2 = function(p3, p4) {}
-const var1 = [];
-function fn3(p5 = {}) {const {arg3} = p5}
+function f3() {
+  const var1 = [];
+  return var1;
+}
 function fn4({ arg4: { arg5, arg_6: arg6 } }) {}
-type Fn5Params = {};
-function fn5({ arg7: { arg8 } }: Fn5Params) {}
 const {
   root_see_all_link_text: rootSeeAllLinkText,
   root_subtitle: rootSubtitle,
   root_title: rootTitle,
-} = cancellationModule || {};
```

Comment:
Removed `cancellationModule` because it's undeclared, but I think we're still testing the same thing we were previously.